packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -47,6 +47,7 @@ export class AtomicSearchInterface {
   @Prop() i18n: i18n = i18next.createInstance();
   @Prop({reflect: true}) language = 'en'; // TODO: make watchable and update i18next language on change
   @Prop({mutable: true}) engine?: Engine;
+  @Prop() enableSearchParameterSerialization = true;
```

Comment:
So, the name is ... meh ? 🤷  
As in, I'm not sure an end user would understand what this does just by looking at the name.


Some proposition:

```
enableRouting
enableURLEncode
enableURLState
enableURLManagement
```

Not sure... 🤔  URL seems like a good word to include there to express what this does ?

Opinion ?

Reply:
Yeah, mentioning `url` in the name would be helpful. Of the suggestions I like `enableUrlState` the best.
Another suggestion: `reflect-state-in-url`